IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.Tests/BaseTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace IntelliTect.TestTools.Selenate.Tests
+{
+    public class BaseTest
+    {
+        protected void TestVoidException()
+        {
+            throw new Exception();
+        }
+
+        protected bool TestReturnException()
+        {
+            throw new Exception();
+        }
+
+        protected void TestVoidNullRefException()
+        {
+            throw new NullReferenceException();
+        }
+
+        protected void TestReturnNullRefException()
+        {
+            throw new NullReferenceException();
+        }
+
+        protected void CheckExceptionsVoidReturn(int secondsToFail = 1, int numberOfDifferentExceptions = 1)
+        {
+            ThrowExceptions(secondsToFail, numberOfDifferentExceptions);
+        }
+
+        protected bool CheckExceptionsBoolReturn(int secondsToFail = 1, int numberOfDifferentExceptions = 1)
+        {
+            ThrowExceptions(secondsToFail, numberOfDifferentExceptions);
+            return true;
+        }
+
+        private void ThrowExceptions(int secondsToFail, int numberOfExceptions)
+        {
+            if (_Timeout == TimeSpan.MinValue)
+            {
+                _Timeout = TimeSpan.FromSeconds(secondsToFail);
+                _Sw.Start();
+            }
+
+            while (_Sw.Elapsed <= _Timeout)
+            {
+                _Attempts++;
```

Comment:
This field only appears to be used inside of this method. It should be converted to a local variable.

Reply:
Same as above. I'll follow up with you on a better way to do this.